frontend/public/components/utils/units.js

```diff
@@ -215,7 +215,7 @@ export const humanizeSeconds = (v, initialUnit, preferredUnit) =>
   humanize(v, 'seconds', true, initialUnit, preferredUnit);
 export const humanizeCpuCores = (v) => {
   const value = v < 1 ? round(v * 1000) : v;
-  const unit = v < 1 ? 'm' : '';
+  const unit = v < 1 ? ' m' : '';
```

Comment:
should we also change units to `cores` and `milicores` according [1780](https://jira.coreos.com/browse/CONSOLE-1780?focusedCommentId=129554&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-129554) ?

Reply:
Could this be made flexible enough to support both? Maybe not right now for time, but in the future?

`# millicores` is probably too long to look natural in the y-axis of a chart, so ` m` or ` c` is fine there, but in other areas (like in tooltips or the other half of a Utilization card) it would be nice to be able to spell out `cores` and `millicores` entirely. "m" is probably very unfamiliar to new Kubernetes users.

![millicores-spelled-out](https://user-images.githubusercontent.com/9122899/68035223-987c7580-fc99-11e9-80fb-315965028b54.png)

Reply:
sure we can! @mareklibra

Reply:
I don't think we should abbreviate cores as `c` since it's not a real abbreviation in k8s. I would say either spell it out fully as `cores` where we have room or leave the unit off to match how cores are specified in the YAML.

+1 to being able to specify a "long" format in the humanize function.